python/examples/structure_map.py

```diff
@@ -11,27 +11,39 @@
 widget in a Jupyter notebook.
 """
 
+# %%
+#
+
```

Comment:
why is this needed?

Reply:
(and similarly for all other examples)

Reply:
To separate the header from the first cell

Reply:
ok, fair enough